PhilipFunkos/src/contact.jsx

```diff
@@ -0,0 +1,18 @@
+import "./contact.css";
+export default function Contact() {
+  return (
+    <div className="contact-page disable">
+      <div className="contact-header">Contact Us</div>
+      <div className="contact-philip">
```

Comment:
You can reuse classes if they have similar styles instead of duplicating code. It should be `contact-block` instead of `contact-phillp` and `contact-angie`.